Package.swift

```diff
@@ -16,10 +16,15 @@ let package = Package(
       targets: ["ComposableArchitecture"]
     ),
   ],
+  dependencies: [
+    .package(url: "https://www.github.com/pointfreeco/swift-case-paths", from: "0.1.0")
```

Comment:
Let's bump to 0.1.1 before merging. Also let's drop the `www`:

```suggestion
    .package(url: "https://github.com/pointfreeco/swift-case-paths", from: "0.1.0")
```